zerocopy-derive/tests/ui/struct.rs

```diff
@@ -15,6 +15,16 @@ fn main() {}
 #[repr(C)]
 struct AsBytes1<T>(T);
 
+#[repr(C, align(4))]
+struct AlignedU32(u32);
```

Comment:
Same here - `derive(AsBytes)`.

Reply:
This is fine to derive because the padding issue will error anyway. Adding this gives the test a higher signal to noise ratio so I think it's a good add.